apps/github-stack/src/github-ineraction/service/sending-email.service.ts

```diff
@@ -59,8 +63,16 @@ export class SendingEmailService {
   }
   async sendMonthSummary(user: User) {
     const summary = user.repositories
-      .map((repo) => `- ${repo.name}`)
-      .join("\n");
+      .map((repo) => {
+        const releases = repo.release.map(
```

Comment:
did you test this method , i don't think it works
Because you are not returning anything to outer map funtion.

I think summary will probably be empty